spec/features/also_available_spec.rb

```diff
@@ -11,6 +11,7 @@
   context 'when a document does not have a searchworks url' do
     it 'does not shows it' do
       visit solr_document_path 'stanford-cg357zz0321'
+      expect(page).to have_content '10 Meter Contours: Russian River Basin, California'
       expect(page).to have_no_link 'SearchWorks'
```

Comment:
These changes are because of a new cop that doesn't like capybara negation matchers being used immediately after page load (presumably because they can give a false positive). I added a trivial matcher to check that the page title is correct before attempting a negation matcher.